integrations/teams/src/channels/outbound.ts

```
@@ -88,11 +88,29 @@ const _makeCard = (card: BotpressCard): Attachment => {
   return CardFactory.heroCard(title, images, buttons, { subtitle })
 }
 
-const _makeDropdownCard = (text: string, options: DropdownOption[]): Attachment => {
-  const choices = options.map((option: DropdownOption) => ({
-    title: option.label,
-    value: option.value,
-  }))
+const _distinctByValue = (choice: { value: string }, index: number, arr: { value: string }[]): boolean => {
+  return arr.findIndex((otherChoice) => otherChoice.value === choice.value) === index
+}
+
+const _makeDropdownCard = (text: string, options: DropdownOption[], logger: bp.Logger): Attachment => {
+  const uniqueChoices = options
+    .map((option: DropdownOption) => ({
+      title: option.label,
+      value: option.value,
+    }))
+    // Hotfix: This exists because some client's code was
+    // duplicating the options everytime the workflow was ran
+    .filter(_distinctByValue)
```

While this addresses the issue for the client, I personally disagree with the idea of cleaning up the data this way since it silently masks the problem and makes it less noticeable for the bot dev.